### PR TITLE
AddHomesWithBestGradesToHomePage

### PR DESCRIPTION
**Pull request by SilviuRusu3**
Added the homes with the best reviews to the Home Page.

### CODE REVIEW

File: HomeFinder/HelpClass/HomeComparer.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HomeFinder.HelpClass
+{
+    public class HomeComparer : IComparer<Home>
+    {
+        
+        //Class used with Sort to sort Homes in reverse order by GeneralGrade
+        public int Compare([AllowNull] Home x, [AllowNull] Home y)
```
Comment (andreisamoila74) [inline HomeFinder/HelpClass/HomeComparer.cs]: let's call x and y better

File: HomeFinder/HelpClass/HomeComparer.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            }
+            else
+            {
+                if (y.GeneralGrade == null)
+                {
+                    return -1;
+                }
+                else
+                {
+                    double difference = (double)x.GeneralGrade - (double)y.GeneralGrade;
```
Comment (andreisamoila74) [inline HomeFinder/HelpClass/HomeComparer.cs]: difference between what? same here, this variable should have a better name

**Review by andreisamoila74 — CHANGES_REQUESTED** [review]

File: HomeFinder/HelpClass/HomeComparer.cs (hunk 1 above)
Comment (SilviuRusu3) [reply HomeFinder/HelpClass/HomeComparer.cs]: I used this class for sorting so I figured it doesn't matter the names of these intermediate variables. To implement custom sorting I searched the Documentation. The variables in the example were called x and y(and x and y were dinosaur objects). I'll change the names to firstHouse and secondHouse.

File: HomeFinder/HelpClass/HomeComparer.cs (hunk 2 above)
Comment (SilviuRusu3) [reply HomeFinder/HelpClass/HomeComparer.cs]: Ok. I'll change the name to gradeDifference

**Pull request merged**